spec/ttdrest/concerns/campaign_spec.rb

```diff
@@ -10,6 +10,22 @@
       let(:budget)      { 1.01 }
       let(:start_date)  { Date.today }
 
+      describe "campaign_flights" do
```

Comment:
Since there is an if, do we need to test the response when not sending it?

Reply:
yes, now tested